platform/linux-generic/odp_classification.c

```diff
@@ -89,8 +102,22 @@ int odp_classification_init_global(void)
 		LOCK_INIT(&pmr->s.lock);
 	}
 
+	queue_grp_shm = odp_shm_reserve("shm_odp_cls_queue_grp_tbl",
+					sizeof(_cls_queue_grp_tbl_t),
+					sizeof(queue_entry_t *), 0);
+
```

Comment:
@muvarov Can Travis detect these sort of _ishm.c warning messages? These should always be flagged as changes shouldn't introduce potential memory leaks.